apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -101,7 +101,7 @@ function LandingPage({
 
   const showGettingStartedBanner =
     !currentYearApplicationId &&
-    workshopsAsParticipant?.length === 0 &&
+    !hasEnrolledInWorkshop &&
```

Comment:
Thank you for simplifying this into a variable that explains what it's actually being used for!